### ISSUE TITLE
Conflicts when trying to install using cabal install

### ISSUE THREAD

**Issue by St3ak91** [bacher09/darkplaces-rcon#3]
![screenshot from 2019-01-17 23-03-31](https://user-images.githubusercontent.com/34294929/51364988-1de8bd80-1aac-11e9-9b19-24e7e2e7dfa4.png)

When try to install darkplaces-rcon-util via cabal install I get a cannot resolve dependencies.
Building from source works though.

**Comment by bacher09** [comment]
Yeah some dependencies are outdated, I updated them in repo but not updated source package on hackage.

**Comment by St3ak91** [comment]
> Yeah some dependencies are outdated, I updated them in repo but not updated source package on hackage.

Can you update the source package? I reinstalled and can't get it to compile manually anymore.
